py-pkg/.gitignore

```diff
@@ -0,0 +1,137 @@
+# From https://github.com/github/gitignore/blob/master/Python.gitignore
```

Comment:
could we just add all this stuff to the main gitignore? What's the benefit of having sub-gitignores?

Reply:
Given that these files are large and highly language-specific, it feels cleaner to keep them separate.